test/unit/diff.js

```diff
@@ -0,0 +1,14 @@
+require('require-self-ref')
+const test = require('ava')
+
+const rssDiff = require('~/src/rss-diff')
+
+const beforeFeed = require('./before_feed.json')
+const afterFeed = require('./after_feed.json')
+const feedDiff = require('./feed_diff.json')
+
+test('test json diffing', t => {
+
```

Comment:
Some extra space here. `standard` wouldn't be too happy about this. btw you should totally add linting (in a separate PR).